src/windows/audio.c

```diff
@@ -381,16 +393,28 @@ void MTY_AudioReset(MTY_Audio *ctx)
 	}
 }
 
-static bool audio_handle_device_change(MTY_Audio *ctx)
+static HRESULT audio_reinit_device(MTY_Audio *ctx)
```

Comment:
Same as my other comment, I intentionally am returning HRESULT so that upon an error the code can be logged.

Reply:
it's highly preferable to return a machine-word-sized error code from any function that returns an error, rather than a boolean, this enables us to keep track of *why* something failed, far after we've left the function.

though we don't do this, it is possible to have a unique 32-bit code for each possible line that errors, in which case simply by seeing the logged error code, you can know on which line the error occurred, even as source changes over time as PRs land.